### ISSUE TITLE
Ask for alt text when uploading an image resource

### ISSUE THREAD

**Issue by christianp** [numbas/editor#519]
When uploading an image as a resource, the author should be asked to provide alt text. This should be editable afterwards. The alt text would be included when the image is inserted in a content area.

This relates to [ATAG 2.0 criterion B.2.3.3](https://www.w3.org/TR/IMPLEMENTING-ATAG20/#sc_b233)
